content/en/security_platform/default_rules/_index.md

```diff
@@ -9,6 +9,10 @@ aliases:
 disable_sidebar: true
 ---
 
-Datadog provides out-of-the-box (OOTB) detection rules to flag attacker techniques and potential misconfigurations so that you can immediately take steps to remediate. Datadog continuously develops new default detection rules, which are automatically imported into your account.
+Datadog provides out-of-the-box (OOTB) detection rules to flag attacker techniques and potential misconfigurations so you can immediately take steps to remediate. Datadog continuously develops new default rules, which are automatically imported into your account or with the Agent. For more information, see the Detection Rules documentation.
 
-Filter by **Logs Detection** to see the Cloud SIEM detection rules, **Workload Security** to see Cloud Security Workload detection rules, **Cloud Configuration** or **Infrastructure Configuration** to see the Cloud Security Posture detection rules.
+Click on the buttons below to filter by different parts of the Datadog Security Platform. OOTB rules are available for [Cloud SIEM][1], [Posture Management][2], which is divided into cloud or infrastructure configuration, [Workload Security][3], and Application Security.
```

Comment:
Miss link to AppSec

Reply:
Intentionally removed as we are publishing this page first. Will add the link once Kari's branch is published with the docs.